### PR TITLE
upd(spotify-client-deb): `1.1.72.439.gc253025e` -> `1.1.80.699.gc3dac…

### PR DESCRIPTION
**Pull request by WRM-42**
…750`

### CODE REVIEW

File: packages/spotify-client-deb/spotify-client-deb.pacscript [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
+gives="spotify-client"
+pkgname="spotify-client"
+repology=("project: spotify")
+breaks="${pkgname}-git ${pkgname}-app ${pkgname}-bin spotify"
+replace="${pkgname}"
+version="1.1.80.699.gc3dac750"
+url="http://repository.spotify.com/pool/non-free/s/${gives}/${gives}_${version}_amd64.deb"
+description="Spotify streaming music client"
+hash="19f80255b89d768969ff9d6a05d90091a30c05f65eabdd968592d47a7754b80e"
+maintainer="TwilightBlood <hwengerstickel@pm.me>"
```
Comment (wizard-28) [inline packages/spotify-client-deb/spotify-client-deb.pacscript:11]: @Henryws, so you're maintaining this in the future right?

File: packages/spotify-client-deb/spotify-client-deb.pacscript [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
-breaks="${pkgname} ${pkgname}-git ${pkgname}-app ${pkgname}-bin spotify"
-version='1.1.72.439.gc253025e'
-url="http://repository.spotify.com/pool/non-free/s/spotify-client/spotify-client_${version}_amd64.deb"
-description='Spotify streaming music client'
-hash='c211a16b6ed6627c4b7d8ca6997376f8345dca638938696933c5495405ce9e16'
-maintainer='TwilightBlood <hwengerstickel@pm.me>'
+name="spotify-client-deb"
+gives="spotify-client"
+pkgname="spotify-client"
+repology=("project: spotify")
```
Comment (wizard-28) [inline packages/spotify-client-deb/spotify-client-deb.pacscript:4]:
```suggestion
repology=("project: spotify" "repo: scoop")
```

Also looks like this is already outdated. So update the pacscript.

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/spotify-client-deb/spotify-client-deb.pacscript (hunk 1 above)
Comment (Henryws) [reply packages/spotify-client-deb/spotify-client-deb.pacscript:11]: Ah, I think I made that at someones request, iirc. No, I don't wish to maintain it as I don't use spotify.

Comment (wizard-28) [reply packages/spotify-client-deb/spotify-client-deb.pacscript:11]: @WRM-42 do you want to take over maintaining this?

**Review by Henryws — APPROVED** [review]
Works

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**